projects/Mallard/src/hooks/__tests__/use-edition-provider.spec.tsx

```diff
@@ -26,6 +28,107 @@ jest.mock('src/services/remote-config', () => ({
     },
 }))
 
+const specialEditions: SpecialEdition[] = [
+    {
+        edition: 'special-edition-expired' as Edition,
+        expiry: new Date(2020, 1, 1),
+        editionType: 'Special',
+        notificationUTCOffset: 1,
+        topic: 'food',
+        title: `Food
+Monthly`,
+        subTitle: 'Store cupboard special: 20 quick and easy lockdown suppers',
+        header: {
+            title: 'Food',
+            subTitle: 'Monthly',
+        },
+        buttonImageUri:
+            'https://media.guim.co.uk/49cebb0db4a3e4d26d7d190da7be4a2e9bd7534f/0_0_103_158/103.png',
+        buttonStyle: {
+            backgroundColor: '#FEEEF7',
+            expiry: {
+                color: '#7D0068',
+                font: 'GuardianTextSans-Regular',
+                lineHeight: 16,
+                size: 15,
+            },
+
+            subTitle: {
+                color: '#7D0068',
+                font: 'GuardianTextSans-Bold',
+                lineHeight: 20,
+                size: 17,
+            },
+            title: {
+                color: '#121212',
+                font: 'GHGuardianHeadline-Regular',
+                lineHeight: 34,
+                size: 34,
+            },
+            image: {
+                height: 134,
+                width: 87,
+            },
+        },
+    },
+    {
+        edition: 'special-edition-notexpired' as Edition,
+        expiry: new Date(3000, 3, 1),
```

Comment:
look! that must be a science fiction edition

Reply:
🤖